.github/workflows/master.yml

```diff
@@ -0,0 +1,75 @@
+on:
+  push:
+    branches:
+      - master
+      - main
+  release:
+    types:
+      - created
+
+name: Publish NPM package
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 16.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16.x
+      - name: Cache node modules
+        id: cache-npm
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-node-modules
+        with:
+          # npm cache files are stored in `~/.npm` on Linux/macOS
+          path: ~/.npm
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - if: ${{ steps.cache-npm.outputs.cache-hit != 'true' }}
+        name: List the state of node modules
+        continue-on-error: true
+        run: npm list
+      - name: Install dependencies
+        run: npm install
+      - name: Prettier check
+        run: npm run check:prettier
+      - name: Check linting
+        run: npm run check:code
+      - name: Build library
+        run: npm run build
+
+  deploy:
+    needs: build
+    runs-on: ubuntu-latest # check if necessary
+    steps:
+      - name: Build storybook
+        run: npm run build:storybook
+      - name: Copy styles from lib
+        run: cp ./lib/*.css ./storybook-static
+      - name: Deploy to GitHub Pages
+        run: |
+          npm init --yes
+          npm install gh-pages --no-save
+
+          mkdir storybook-static/.circleci
+          touch storybook-static/.circleci/config.yml
```

Comment:
is this `.circleci` necessary?

Reply:
I believe it's not necessary anymore, removed!